system/churchcore/classes/qqFileUploader.class.php

```diff
@@ -12,7 +12,7 @@ class qqFileUploader {
    * create qqUploadedFileXhr or qqUploadedFileForm depending of variable set (get or file)
    *
    * @param array $allowedExtensions
-   * @param number $sizeLimit
+   * @param int $sizeLimit
    */
   function __construct(array $allowedExtensions = array(), $sizeLimit = 10485760) {
```

Comment:
I agree, how about replacing the default value with "null", and do an isset()-check in the function?